Formula/r.rb

```diff
@@ -95,12 +128,26 @@ def post_install
 
   test do
     assert_equal "[1] 2", shell_output("#{bin}/Rscript -e 'print(1+1)'").chomp
-    assert_equal ".dylib", shell_output("#{bin}/R CMD config DYLIB_EXT").chomp
-    assert_equal "[1] \"aqua\"",
-                 shell_output("#{bin}/Rscript -e 'library(tcltk)' -e 'tclvalue(.Tcl(\"tk windowingsystem\"))'").chomp
+    dylib_ext = ".dylib"
+    on_linux do
+      dylib_ext = ".so"
+    end
```

Comment:
Is `shared_library` available in `test do` blocks?

Reply:
It is, thanks for the catch!